MPF.Core/Modules/Redumper/Parameters.cs

```diff
@@ -363,6 +363,9 @@ public override void GenerateSubmissionInfo(SubmissionInfo info, Options options
                         }
 
                         info.Extras!.PIC = GetPIC($"{basePath}.physical", trimLength) ?? string.Empty;
+
+                        var di = InfoTool.GetDiscInformation($"{basePath}.physical");
+                        info.SizeAndChecksums!.PICIdentifier = InfoTool.GetPICIdentifier(di);
```

Comment:
I think this can be put before the previous line and then the PIC object that comes out can be passed into the `GetPIC` method instead.

Reply:
`GetDiscInformation` deserializes the PIC, so I think it would need re-serialising before passing to GetPIC, which defeats the purpose.

Reply:
Alright then, good enough.